pkg/selinux/selinux.go

```diff
@@ -409,19 +409,21 @@ func GetLxcContexts() (processLabel string, fileLabel string) {
 	}
 
 	if processLabel == "" || fileLabel == "" {
-		return "", ""
+		return "", "", nil
```

Comment:
Yeah I think that's reasonable. It's valid for selinux to be enabled and for us to have no lxc_contexts - there's no standard for selinux policy. In that case we should just continue without it.